scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -399,6 +455,9 @@ def validate_assets(input_dirs: List[Path],
                 if check_build_context:
                     error_count += validate_build_context(asset_config.extra_config_as_object())
 
+            if asset_config.type == assets.AssetType.PROMPT:

```

Comment:
let's include EvaluationResult type here too

Reply:
Added validation for eval results.  It uses the same list as prompts except prompt has modality set as required.